amf-client/shared/src/test/scala/amf/maker/InvalidsRefElementTest.scala

```diff
@@ -26,7 +26,7 @@ class InvalidsRefElementTest extends AsyncFunSuite with CompilerTestBuilder {
     super.defaultConfig.withErrorHandlerProvider(() => IgnoringErrorHandler)
 
   test("Invalid link to Response with ast") {
-    build("file://amf-client/shared/src/test/resources/invalids/error-response.oas", Oas20YamlHint)
+    build("file://amf-client/shared/src/test/resources/invalids/error-response.yaml", Oas20YamlHint)
```

Comment:
InvalidsRefElementTest: extension of file was modified in a previous commit to avoid having to explicitly pass content type.